### DISCUSSION TITLE
What is your favorite games?

### DISCUSSION THREAD

**Discussion by tmksobue** [NAIST-SE/AttractingContributors#1116] [2024 Attraction Contributions]
I would like to know games you like! 
Please edit here and send me your pull requests⤵
Click: https://github.com/tmksobue/AttractingContributors/blob/main/README.md

**Comment by mustafizur-r** [comment]
Hello [@tmksobue](https://github.com/tmksobue) , I recently made a contribution to your project , I have added “Cricket". 
could you please contribute to my as well ? Following is the link to [my repository](https://github.com/MustafizurRU/AttractingContributors-My_Favorite_Sci-Fi_Books) . Thanks in advance.

**Reply by tmksobue** [reply]
Sorry for the late reply. I've added an item to your repository. Thank you for contributing to mine :)

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Reply by tmksobue** [reply]
Sorry for the late reply. I've added an item to your repository. Thank you for contributing to mine!

**Comment by thapakorn604** [comment]
Hi, I haved added my game to your list https://github.com/tmksobue/AttractingContributors/pull/12
if you don't mind please help me add a song to my topic : ) https://github.com/NAIST-SE/AttractingContributors/discussions/1115

**Reply by tmksobue** [reply]
Sorry for the late reply, and thank you for contributing my repository. I've added an item to your repository.

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon

**Reply by tmksobue** [reply]
Sorry for the late reply. Thank you for contributing my repository, and I've added an item to yours!
